src/features/Login/index.jsx

```diff
@@ -1,12 +1,67 @@
-import React, { useState } from 'react';
-import { Form, Input, Button, Checkbox } from 'antd';
-import "./index.scss"
+import React, { useState } from "react";
+import { Form, Input, Button, Row, Col, Space } from "antd";
+import { useHistory } from "react-router-dom"
+import "./index.scss";
+import logo from "../../assets/images/logo.png";
```

Comment:
Can define a file constant for image, and import to using

Reply:
Fixed at commit [fd36968|https://github.com//pull/1]